src/istio/control/http/request_handler_impl.cc

```diff
@@ -66,6 +66,7 @@ CancelFunc RequestHandlerImpl::Check(CheckData* check_data,
                                      CheckDoneFunc on_done) {
   // Forwarded attributes need to be stored regardless Check is needed
   // or not since the header will be updated or removed.
+  AddCheckAttributes(check_data);
```

Comment:
Wayne, see API https://github.com/istio/api/blob/master/mixer/v1/config/client/client_config.proto#L87 for better or worse. The goal is to restore 1.0 behavior since customers are affected.

Reply:
OK, approved